ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -488,18 +489,26 @@ aaa_server_groups:
 ```yaml
 aaa_authentication:
   login:
-    default: < group | local | none >
-    serial_console: < group | local | none >
+    default: < group < name >  | local | none >
+    serial_console: < group < name >  | local | none >
   dot1x:
-    default: < group | local | none >
+    default: < group < name > | local | none >
+  policies:
+    on_failure_log: < true | false >
+    on_success_log: < true | false >
+    local:
+        allow_nopassword: < false | true (default) >
 ```
 
 ### AAA Authorization
 
 ```yaml
 aaa_authorization:
-  exec_default: < group | local | none >
+  exec_default: < group < name > | local | none >
```

Comment:
Should be `< group name >` and not `< group < name >`

Reply:
Fixed this and the others, I think what I wanted to emphasise here is to have group part of the command and then have a name for the group.